geopandas/io/arrow.py

```diff
@@ -57,15 +57,17 @@ def _create_metadata(df):
     for col in df.columns[df.dtypes == "geometry"]:
         series = df[col]
         column_metadata[col] = {
-            "crs": series.crs.to_wkt() if series.crs else None,
             "encoding": "WKB",
+            "geometry_type": list(Series(series.geom_type.unique()).dropna()),
```

Comment:
> It seems like this should also be sorted, if for no other reason than for consistency in tests.

Ah, yes. Sort it alphabetically, or by logic? (eg ["Polygon", "MultiPolygon"] instead of the other way around). For tests it of course doesn't really matter, as long as it is sorted.

Reply:
I was thinking alphabetically, but that is probably too narrow vs how this might be used in practice where it is actually used.  E.g., multi before singular.  Sorting seems an issue for the spec to take up.

Maybe instead of sorting here, just sort when doing comparison in test?